tests/components/composer.test.ts

```diff
@@ -1339,154 +868,157 @@ describe("composer", () => {
     expect(model.getters.getEditionMode()).toBe("inactive");
 
     // click on the modified cell C8
-    await clickCell(model, "C8");
+    selectCell(model, "C8");
+    await nextTick();
     expect(getSelectionAnchorCellXc(model)).toBe("C8");
     expect(model.getters.getEditionMode()).toBe("inactive");
   });
 
-  test("Add a character changing the edition mode to waitingForRangeSelection correctly renders the composer", async () => {
-    await typeInComposerGrid("=sum(4");
+  test("Add a character changing the edition mode to 'selecting' correctly renders the composer", async () => {
+    await typeInComposer("=sum(4");
     expect(cehMock.selectionState.isSelectingRange).toBeFalsy();
-    await typeInComposerGrid(",");
+    await typeInComposer(",");
     expect(cehMock.selectionState.isSelectingRange).toBeTruthy();
   });
 
-  test("f4 shortcut on cell symbol", async () => {
-    composerEl = await typeInComposerGrid("=A1");
-    model.dispatch("CHANGE_COMPOSER_CURSOR_SELECTION", { start: 1, end: 1 });
-    await keyDown("F4");
-    expect(model.getters.getCurrentContent()).toBe("=$A$1");
-    await keyDown("F4");
-    expect(model.getters.getCurrentContent()).toBe("=A$1");
-    await keyDown("F4");
-    expect(model.getters.getCurrentContent()).toBe("=$A1");
-    await keyDown("F4");
-    expect(model.getters.getCurrentContent()).toBe("=A1");
-  });
+  describe("Loop through reference combinations (AKA F4 shortcut)", () => {
```

Comment:
all those tests could be converted to plugin tests.
Except one to test the integration with the plugin. (there's really only one line in the composer)